### ISSUE TITLE
Bad timestamp at disappear_time

### ISSUE THREAD

**Issue by Fiofant** [AHAAAAAAA/PokemonGo-Map#603]
In http://0.0.0.0:5001/pokemons i see disappear_time:
```
[
  {
    "disappear_time": 1469047103908, 
    "encounter_id": "NzExODIwNjgwNjE0ODk2NjY4NQ==", 
    "latitude": 67.9378975620991, 
    "longitude": 32.92954718254374, 
    "pokemon_id": 96, 
    "pokemon_name": "Drowzee", 
    "spawnpoint_id": "443175464c9"
  }
] 
```
It timestamp (1469047103908) is  Wed, 01 Apr 48522 23:58:28 GMT;
If remove the last three digits (1469047103) is (Wed, 20 Jul 2016 20:38:23 GMT)

**Comment by mkorsak** [comment]
Correct, disappear time doesn't seem to be working. No countdown, no expire.

**Comment by wcastello** [comment]
#578 for countdown and expire, and the timestamp you see is * 1000 in order to work in Javascript's Date(). Divide by 1000 and you'll see it's correct.

**Comment by MonkeyLeeT** [comment]
Yeah in the rewrite they forgot that it was milliseconds. Will be fixed once #578 is merged.
